tests/doghouse/test_delta_engine.py

```
@@ -0,0 +1,53 @@
+import datetime
+from doghouse.core.domain.blocker import Blocker, BlockerType
+from doghouse.core.domain.snapshot import Snapshot
+from doghouse.core.services.delta_engine import DeltaEngine
+
+def test_compute_delta_no_changes():
+    engine = DeltaEngine()
+    blocker = Blocker(id="1", type=BlockerType.UNRESOLVED_THREAD, message="msg")
+    
+    baseline = Snapshot(
+        timestamp=datetime.datetime(2026, 1, 1),
```

Ruff (0.15.7)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)